code/modules/projectiles/guns/bang.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/weapon/gun/bang
+	name = "some shitty-ass gun"
```

Comment:
The name might be better off as something like "Discount revolver" if you're going for prank value.

Reply:
It's a base class.